File: Iniciante/URI 1009.pas
var
    nome : string;
	salarioFixo : double;
	vendas : double;
	total : double;

begin
    read(nome);
    read(salarioFixo);
	read(vendas);
	
    total := ((vendas * 15) / 100) + salarioFixo;
    
	writeln('TOTAL = R$ ', total:0:2);
end.

File: Iniciante/URI 1009.clj
(def nome (read-string (read-line)))
(def salarioFixo (read-string (read-line)))
(def vendas (read-string (read-line)))
(def total (+ (/ (* vendas 15.0) 100.0) salarioFixo))

(printf "TOTAL = R$ %.2f\n" total)